view/frontend/web/js/view/delivery-options.js

```diff
@@ -228,11 +238,17 @@ define(
 
         checkout.convertDeliveryOptionsToShippingMethod(event.detail, {
           onSuccess: function(response) {
-            if (!response.length) {
+            if (! response.length) {
```

Comment:
die spatie hoeft niet in js, gebruik je eslint? (er vanuitgaande dat dat erin zit...)